src/components/sign-in/sign-in.tsx

```diff
@@ -96,11 +98,11 @@ const SignIn = ({ dark, position }: Props) => {
       {!provider && (
         <ConnectButton
           type={dark ? 'primary' : 'secondary'}
-          size="sm"
+          size="xxs"
           md={{ size: 'md' }}
           className={styles.connectButton}
         >
-          Connect Wallet
+          <span className={styles.connectButtonLabel}>Connect {isDesktop ? 'Wallet' : ''}</span>
```

Comment:
Not a big deal, but we can do without the hanging space after Connect

```suggestion
          <span className={styles.connectButtonLabel}>{isDesktop ? 'Connect Wallet' : 'Connect'}</span>
```

Reply:
Yeah, that's better, thanks!